docs/en/upgrade.md

```diff
@@ -55,7 +55,15 @@ You can view the status of the stack in the AWS CloudFormation console in the **
 
 You can view the status of the pipeline in the solution console in the **Status** column. After a few minutes, you can receive an Active status.
 
-## Post-Upgrade Actions (only applicable to data modeling enabled)
+### Ingestion
+
+Due to AWS no longer supporting Launch Configurations in Auto Scaling Groups, we will use Launch Template instead. After upgrading the data ingestion module to v1.2.0, please follow below steps to replace the EC2 instances used by Amazon ECS with the new Launch Template configuration:
+
+1. Manually increase the desired number of tasks in Amazon ECS.
+2. Once the newly added Amazon ECS tasks have started successfully, manually stop the old tasks.
+3. Manually terminate the old EC2 instances.
+
+### Data Modeling
 
 ### Upgrade the Data Schema and Out-of-the-box Dashboards
```

Comment:
update to level 4 for Upgrade the Data Schema and Out-of-the-box Dashboards / Migrate the Existing Data (only applicable when upgrading from a version earlier than v1.1.6)